lib/modules/datasource/dotnet-version/schema.ts

```diff
@@ -26,17 +26,22 @@ export const DotnetSdkReleases = z
   .object({
     releases: LooseArray(
       ReleaseBase.extend({
-        sdk: ReleaseDetails,
```

Comment:
Was this a bug and fields doesn't exist? Or it exists but is superseded by sdks?

Reply:
this exists, but same version is also contained in `sdks` array, so i removed the duplicate